### PR TITLE
MINOR: Add the schedule, fix label, fix defaults

### PR DESCRIPTION
**Pull request by mumrah**
Small follow-up to #13827 

* Add the cron schedule to run nightly
* Use the correct property for PR labels
* Add defaults for the variables in the action (I tested this on mumrah/kafka)

### CODE REVIEW

**Comment by mumrah** [comment]
attn @jlprat @dajac

**Comment by mumrah** [comment]
The dry-run worked fine, so I did a small run limited to 30 PRs:

```
Statistics:
Processed PRs: 117
New stale PRs: 7
Added PRs labels: 7
Added PRs comments: 7
Fetched items: 200
Fetched items events: 7
Fetched items comments: 7
Operations performed: 30
```

Here are the 7 PRs it found https://github.com/apache/kafka/pulls?q=is%3Aopen+is%3Apr+label%3AStale

**Review by jlprat — APPROVED** [review]
LGTM thanks @mumrah

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -40,13 +40,16 @@ jobs:
...
           days-before-stale: 90
           days-before-close: -1
-          stale-issue-label: 'stale'
+          stale-pr-label: 'stale'
           stale-pr-message: >
             This PR is being marked as stale since it has not had any activity in 90 days. If you
             would like to keep this PR alive, please ask a committer for review. If the PR has 
             merge conflicts, please update it with the latest from trunk (or appropriate release branch)
             
+            If this PR is no longer valid or desired, please feel free to close it. If no activity
```
Comment (jlprat) [inline .github/workflows/stale.yml:53]: Ah, good addition!

**Review by jlprat — APPROVED** [review]
LGTM'ing it again

**Pull request merged**